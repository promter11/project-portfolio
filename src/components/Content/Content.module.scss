.main {
  width: 100%;
  padding-left: 32rem;
  font-family: "Public Sans", sans-serif;
}
