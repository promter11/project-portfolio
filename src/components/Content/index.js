import React from "react";

import Routes from "../../routes";

import styles from "./Content.module.scss";

const Content = ({ language }) => {
  return (
    <main className={styles.main}>
      <Routes language={language} />
    </main>
  );
};

export default Content;
