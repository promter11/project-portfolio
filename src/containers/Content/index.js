import { connect } from "react-redux";

import { getLanguage } from "../../store/selectors/LanguageSelectors";

import Content from "../../components/Content";

const mapStateToProps = (state) => {
  return {
    language: getLanguage(state),
  };
};

export default connect(mapStateToProps)(Content);
