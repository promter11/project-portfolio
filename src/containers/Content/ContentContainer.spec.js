import configureStore from "redux-mock-store";
import { shallow } from "enzyme";

import Content from "./index";

const configureMockStore = configureStore();

const initialState = {
  LanguageReducer: {
    defaultLanguage: {
      language: "RU",
      icon: "russia.svg",
    },
  },
};

const store = configureMockStore(initialState);

describe("Content container", () => {
  it("should renders correctly with props", () => {
    const wrapper = shallow(<Content store={store} />).dive();

    expect(wrapper).toMatchSnapshot();
  });
});
