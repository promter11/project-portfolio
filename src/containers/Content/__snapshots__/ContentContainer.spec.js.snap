// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Content container should renders correctly with props 1`] = `
<Content
  dispatch={[Function]}
  language="RU"
  store={
    Object {
      "clearActions": [Function],
      "dispatch": [Function],
      "getActions": [Function],
      "getState": [Function],
      "replaceReducer": [Function],
      "subscribe": [Function],
    }
  }
/>
`;
